src/Microsoft.SqlTools.ServiceLayer/DacFx/DacFxService.cs

```diff
@@ -228,9 +228,18 @@ private void ExecuteOperation(DacFxOperation operation, DacFxParams parameters,
                 try
                 {
                     TaskMetadata metadata = TaskMetadata.Create(parameters, taskName, operation, ConnectionServiceInstance);
+                    if (operation.GetType() == typeof(ExtractOperation) || operation.GetType() == typeof(ExportOperation))
+                    {
+                        // show package file path in task viewlet instead of server and database name
+                        metadata.DatabaseName = null;
+                        metadata.ServerName = parameters.PackageFilePath;
```

Comment:
I'm not a fan of overloading the field like this - what happens if someone on the ADS side tries to use this expecting it to be a valid server name all the time? If we're going to do this I'd rather see a new field added, something like `TargetDestination` (since it should be generic across many operations)

Reply:
Yeah that's makes sense. How should ADS taskRederer handle this new field? Show the TargetDestination if there aren't values for server and database? Or should it have priority over those?

Reply:
I'd say default to TargetDestination and if it's undefined/empty then fall back to what it was previously.